src/hypervisor.c

```diff
@@ -52,35 +52,9 @@ cc_oci_expand_net_cmdline(struct cc_oci_config *config) {
          * <device>:<autoconf>:<dns0-ip>:<dns1-ip>
 	 */
 
-	/* TODO: Use cloud-init based network init
-	 * For now, just pick the first interface
-	 */
-	struct cc_oci_net_if_cfg *if_cfg = NULL;
-	struct cc_oci_net_ipv4_cfg *ipv4_cfg = NULL;
-
-	if_cfg = (struct cc_oci_net_if_cfg *)
-		g_slist_nth_data(config->net.interfaces, 0);
-
-	if (if_cfg == NULL) {
-		goto out;
-	}
-
-	ipv4_cfg = (struct cc_oci_net_ipv4_cfg *)
-		g_slist_nth_data(if_cfg->ipv4_addrs, 0);
-
-	if (ipv4_cfg == NULL){
-		goto out;
-	}
-
-	return ( g_strdup_printf("ip=%s::%s:%s:%s:%s:off::",
-		ipv4_cfg->ip_address,
+	return ( g_strdup_printf("ip=:::%s:::%s::off::",
 		config->net.gateway,
```

Comment:
This function needs a check to ensure `config != NULL` before dereferencing. Also, I'd add checks on `gateway` and `hostname` to avoid `(null)` being returned as part of the string on error.